stderr = fopen("/dev/null", "w") leads to
wget_css_fuzzer.c:106:9: error: lvalue required as left operand of assignment

--- wget-1.19.5/fuzz/wget_css_fuzzer.c	2018-05-14 11:01:05.990377852 +0000
+++ wget-1.19.5/fuzz/wget_css_fuzzer.c	2018-05-14 11:26:11.684401826 +0000
@@ -82,6 +82,8 @@
 
 int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 {
+	int copy = -1 , fd = -1;
+
 	if (size > 4096) // same as max_len = ... in .options file
 		return 0;
 
@@ -91,9 +93,10 @@
 		.document_file = NULL,
 	};
 
-	FILE *bak = stderr;
-	stderr = fopen("/dev/null", "w");
-
+	copy = dup(STDERR_FILENO);
+	fd = open("/dev/null", O_WRONLY);
+	dup2(fd, STDERR_FILENO);
+	
 	if (setjmp(jmpbuf))
 		goto done;
 
@@ -102,8 +105,9 @@
 	free((void *) ctx.parent_base);
 
 done:
-	fclose(stderr);
-	stderr = bak;
+	close(fd);
+	close(STDERR_FILENO);
+	dup2(copy, STDERR_FILENO);
 
 	return 0;
 }
--- wget-1.19.5/fuzz/wget_cookie_fuzzer.c.~1~	2018-05-05 23:46:22.000000000 +0000
+++ wget-1.19.5/fuzz/wget_cookie_fuzzer.c	2018-05-14 12:04:27.083142620 +0000
@@ -20,6 +20,9 @@
 #include <config.h>
 
 #include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
 #include <dirent.h> // opendir, readdir
 #include <stdint.h> // uint8_t
 #include <stdio.h>  // fmemopen
@@ -71,6 +74,7 @@
 	FILE *bak;
 	struct cookie_jar *cookie_jar;
 	char *set_cookie;
+        int copy = -1, fd = -1;
 
 	if (size > 1024) // same as max_len = ... in .options file
 		return 0;
@@ -79,8 +83,9 @@
 	memcpy(set_cookie, data, size);
 	set_cookie[size] = 0;
 
-	bak = stderr;
-	stderr = fopen("/dev/null", "w");
+	copy = dup(STDERR_FILENO);
+	fd = open("/dev/null", O_WRONLY);
+	dup2(fd, STDERR_FILENO);
 
 	cookie_jar = cookie_jar_new();
 	cookie_handle_set_cookie(cookie_jar, "x", 81, "p", set_cookie);
@@ -88,8 +93,9 @@
 	cookie_handle_set_cookie(cookie_jar, "x", 80, "p/d/", set_cookie);
 	cookie_jar_delete(cookie_jar);
 
-	fclose(stderr);
-	stderr = bak;
+	close(fd);
+	close(STDERR_FILENO);
+	dup2(copy, STDERR_FILENO);
 
         free(set_cookie);
 
--- wget-1.19.5/fuzz/wget_ftpls_fuzzer.c.1	2018-05-14 12:39:02.068264596 +0000
+++ wget-1.19.5/fuzz/wget_ftpls_fuzzer.c	2018-05-14 12:41:07.696460673 +0000
@@ -20,6 +20,8 @@
 #include <config.h>
 
 #include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
 #include <dirent.h> // opendir, readdir
 #include <stdint.h> // uint8_t
 #include <stdio.h>  // fmemopen
@@ -73,12 +75,14 @@
 {
 	FILE *fp, *bak;
 	struct fileinfo *fi;
+	int copy = -1, fd = -1;
 
 	if (size > 4096) // same as max_len = ... in .options file
 		return 0;
 
-	bak = stderr;
-	stderr = fopen("/dev/null", "w");
+	copy = dup(STDERR_FILENO);
+	fd = open("/dev/null", O_WRONLY);
+	dup2(fd, STDERR_FILENO);
 
 	fp = fmemopen((void *) data, size, "r");
 	if (!fp) return 0;
@@ -100,8 +104,9 @@
 
 	fclose(fp);
 
-	fclose(stderr);
-	stderr = bak;
+        close(fd);
+        close(STDERR_FILENO);
+        dup2(copy, STDERR_FILENO);
 
 	return 0;
 }
